README.md

```diff
@@ -2,44 +2,44 @@
 
 [Docker](https://www.docker.com/) and [Boot2Docker](http://boot2docker.io/) are
 awesome for running containers on OS X, but if you try to use them to do
-iterative development by mounting a source folder from OS X into your Docker 
```

Comment:
My editor removes whitespace at the end of files.  If that's a problem (i.e. mucking up the commit history), I can undo and push a less "changed" version.